content/en/docs/01/labs/11.md

```diff
@@ -25,8 +26,9 @@ tar fvxz node_exporter-1.0.1.linux-amd64.tar.gz
 ```
 
 The Prometheus configuration part can be solved by either using a static configuration **or** by using the file service discovery mechanism:
-
-### Option 1: Static config
+<!-- markdownlint-disable MD036 -->
```

Comment:
I wouldn't use those

Reply:
the alternative is to disable the right Navigationbar or to use 4. Level Headings which will trigger MD001 and maybe others... Or do not use Headings in Solutions at all

Reply:
I see, let me quickly have a look at it.

Reply:
I'd rather find general solution (we might need this in other trainings as well) to disable the right navigation for solutions, than disabling the linting. It adds unnecessary boilerplate code and it's quite a hacky solution.

Reply:
let me know if you have such a solution and I will adapt this MR

Reply:
Are you ok with it, if we switch back to h3 and merge, for tomorrows session, to not have a longliving PR? then back back to it, once we have a general solution?

Reply:
sry, missed your last comment, thanks for the change